sdk/core/azure-core/checkstyle-suppressions.xml

```diff
@@ -1,40 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE suppressions PUBLIC
-    "-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN"
-    "https://checkstyle.org/dtds/suppressions_1_2.dtd">
+<!DOCTYPE suppressions PUBLIC "-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN" "https://checkstyle.org/dtds/suppressions_1_2.dtd">
+<!-- This file is generated by the /eng/scripts/linting_suppression_generator.py script. -->
 
 <suppressions>
-  <!-- Suppress VisibilityModifier in MemberConverterImplTests and FieldBuilderTests.-->
-  <suppress checks="VisibilityModifier" files="com.azure.core.implementation.jackson.MemberNameConverterImplTests.java" />
-
-  <!-- Jackson adapter is allowed to expose external dependencies -->
-  <suppress checks="com.azure.tools.checkstyle.checks.ExternalDependencyExposedCheck"
-            files="com.azure.core.util.serializer.JacksonAdapter"/>
-
-  <!-- Logger class suppression -->
-  <suppress checks="com.azure.tools.checkstyle.checks.GoodLoggingCheck" files="(ClientLogger|ClientLoggerAccessHelper|DefaultLogger|LoggingEventBuilder).java"/>
-
-  <suppress checks="com.azure.tools.checkstyle.checks.ThrowFromClientLogger"
-            files="com.azure.core.implementation.jackson.(MemberNameConverterImpl|ObjectMapperFactory|XmlMapperFactory).java"/>
-  <suppress checks="com.azure.tools.checkstyle.checks.ThrowFromClientLogger" files="com.azure.core.implementation.(AccessTokenCache|AccessibleByteArrayOutputStream).java"/>
-  <suppress checks="com.azure.tools.checkstyle.checks.ThrowFromClientLogger" files="com.azure.core.util.logging.LoggingEventBuilder.java"/>
-
-  <!-- Exception is logged in an earlier line. -->
-  <suppress checks="com.azure.tools.checkstyle.checks.ThrowFromClientLogger" files="com.azure.core.http.policy.HttpLoggingPolicy.java"/>
-
-  <!-- Don't log in SyncRestProxy to match AsyncRestProxy. -->
```

Comment:
It would be nice to keep the comments. How is the linting supressions generator script creating these suppressions?

Reply:
The changes here are going from a handwritten to a generated file. I could look into having the generation maintain any existing matches, but I don't think that's worth the effort.